Tasks/ACRTaskV0/utils.ts

```diff
@@ -0,0 +1,134 @@
+import tl = require("azure-pipelines-task-lib/task");
+import * as acrTaskRequest from "./acrtaskrequestbody";
+import * as imageUtils from "docker-common-v2/containerimageutils";
+import * as pipelineUtils from "docker-common-v2/pipelineutils";
+import { Resources } from 'azure-arm-rest-v2/azure-arm-resource';
+import { AzureEndpoint } from 'azure-arm-rest-v2/azureModels';
+import { AcrTask, ACRRegistry } from "./acrtaskclient";
+
+export function getContextPath(): string
+{
+    var repoUri = tl.getVariable("Build.Repository.Uri");
```

Comment:
This could be difficult to target if there are multiple repos targetted in a release definition. Instead, you could take a github endpoint as an input.

Reply:
have taken github connection, repo url and branch as input